cosmos_test.go

```diff
@@ -1268,23 +1268,20 @@ func TestWaitForRetry_Abort(t *testing.T) {
 
 func TestHandleTimeout(t *testing.T) {
 	// GIVEN
-	cosmos := cosmosImpl{
-		logger:       zerolog.New(os.Stdout).Level(zerolog.DebugLevel),
-		retryTimeout: time.Millisecond * 50,
-	}
-	done := make(chan bool)
+	retryTimeout := time.Millisecond * 50
+	done := make(chan struct{})
 	defer close(done)
 
 	timedOut := false
 	mu := sync.Mutex{}
 
 	// WHEN
 	go func() {
-		timedOutChan := cosmos.handleTimeout(done)
+		timedOutChan := handleTimeout(done, retryTimeout, zerolog.Nop())
 
-		timeOut := <-timedOutChan
+		<-timedOutChan
 		mu.Lock()
-		timedOut = timeOut
+		timedOut = true
```

Comment:
now timedOut is always true whether something was sent on the timedOutChan or not